docs/sources/panels/visualizations/gauge-panel.md

```diff
@@ -0,0 +1,39 @@
++++
+title = "Gauge panel"
+description = "Gauge panel docs"
+keywords = ["grafana", "gauge", "gauge panel"]
+type = "docs"
+aliases = ["/docs/grafana/latest/features/panels/gauge/"]
+weight = 300
+draft = "true"
++++
+
+# Gauge panel
+
+Gauge is a single value panel that can repeat a gauge for every series, column or row.
+
+{{< docs-imagebox img="/img/docs/v66/gauge_panel_cover.png" max-width="1025px" >}}
+
+## Data and field options
+
+Gauge visualizations allow you to apply:
+- Data transformations
+- Field settings
+- Field overrides
+- [Thresholds]({{< relref "../thresholds.md">}})
+- Value mappings
+
+## Display options
+
+Use the following options to refine your visualization:
+
+- **Show -** Choose how Grafana displays your data.
+  - **Calculate -** Show a calculated value based on all rows.
+  - **All values -** Show a separate stat for every row. If you select this option, then you can also select a **Limit**, or the maximum number of rows to display.
+- **Value -** Select a reducer function that Grafana will use to reduce many fields to a single value. Click the **Value** list to see functions and brief descriptions.
+- **Orientation -** Choose a stacking direction.
+  - **Auto -** Grafana selects what it thinks is the best orientation.
```

Comment:
Think here's something wrong with the hierarchy.

When you select `Calculate` you will get the `Value` option.
When you select `All value` you will get Limit option

Value is duplicated ^^ and indented unnecessary (the first occurrence)